javascript/javascript1/week2/fullnameformal.js

```diff
@@ -0,0 +1,34 @@
+function getFullname (firstname, surname, useFormalName, gender){
+if (useFormalName) { 
+    if (gender==="man") {
+    console.log(`lord ${firstname} ${surname}`)   
+} else if(gender==="woman") {
+    console.log(`lady ${firstname} ${surname}`)
+}else{
+    console.log(`${firstname} ${surname}`)
+}   
+}
+}
+
+
+getFullname ('Benjamin', 'Hughes', true, "man");
+
+
+
+
+const fullname1 = getFullname('Hanaa','Hussein', true, "woman");
+const fullname2 = getFullname('Sarah', 'Amouri',true);
+
```

Comment:
Check `getFullname("", "");` `getFullname(true, true);` what should be returned?

Reply:
I didnt get any result

Reply:
And what do you think it should be returned?